Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM       node:16 as builder
-RUN        npm i -g --unsafe-perm @subql/node@0.23.1
+RUN        npm i @subql/node@0.31.1 --location=global
 
 FROM       node:16-alpine
 ARG        SORA_ENDPOINT
+ENV        NODE_VERSION=16.14.2
+ENV        YARN_VERSION=1.22.18
 ENV        TZ utc
-RUN        apk add --no-cache tini git
+RUN        apk add --no-cache tini git curl
```

Comment:
Why do you need curl?

Reply:
for healthcheck

Reply:
you can use something like https://github.com/soramitsu/nbc-deploy/blob/develop/fastcore/docker-compose.yml#L8

Reply:
and why do you need a git?

Reply:
edited